File: replacer-backend/src/main/java/es/bvalero/replacer/review/dto/ReviewSectionDto.java
package es.bvalero.replacer.review.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import javax.validation.constraints.NotNull;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.lang.NonNull;

@Schema(description = "Section of a page to review")
@Data
@NoArgsConstructor
public class ReviewSectionDto {

    @Schema(description = "Section ID", required = true, example = "1")
    @NotNull
    int id;

    @Schema(description = "Section title", required = true, example = "Biografía")
    @NonNull
    @NotNull
    String title;
}

File: replacer-frontend/src/app/api/models/review-section-dto.ts
/* tslint:disable */
/* eslint-disable */

/**
 * Section of a page to review
 */
export interface ReviewSectionDto {

  /**
   * Section ID
   */
  id: number;

  /**
   * Offset of the section with the page content
   */
  offset: number;

  /**
   * Section title
   */
  title: string;
}
